doc/languages-frameworks/php.section.md

```diff
@@ -1,10 +1,8 @@
 # PHP
```

Comment:
There is a pandoc extension, not sure if enabled by default, that allows defining ids. All headings need to have IDs specified to maintain permanent reference.

```suggestion
# PHP {#sec-language-php}
```

Reply:
Seems to work, added to all headings in the document :)